collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml

```diff
@@ -8,6 +8,7 @@ Description: |
 
 Documentation: "[Product information](https://resources.marine.copernicus.eu/?option=com_csw&view=details&product_id=OCEANCOLOUR_ATL_CHL_L4_NRT_OBSERVATIONS_009_037)"
 Image: oceancolour-atl-chl-l4-nrt-observations-009-037/oceancolour-atl-chl-l4-nrt-observations-009-037.png
+EDCBrowser: https://browser.eurodatacube.com/?zoom=6&lat=54&lng=-4&collectionId=oceancolour_atl_chl_l4_nrt_observations_009_037&layerId=CHL%20visualization&type=sentinel-hub-edc&fromTime=2020-12-31T00%3A00%3A00.000Z&toTime=2021-12-31T00%3A00%3A00.000Z
```

Comment:
As far as I see, this is not a SH collection and can therefore not be displayed in the EDC Browser in the way specified here. Either update link to something that displays data or remove it.

Reply:
Can't visualise the data so remove the link